esy.json

```diff
@@ -21,7 +21,8 @@
     "refmterr": "*"
   },
   "esy": {
-    "build": [["refmterr", "dune", "build", "-p", "#{self.name}"]],
+    "build": [["dune", "build", "-p", "#{self.name}"]],
+    "buildDev": [["refmterr", "dune", "build", "-p", "#{self.name}"]],
```

Comment:
Sorry one more thing that I forgot. In development you don't wanna compile with the release profile. Instead the command should be:

`refmterr dune build --promote-install-files --root . --only-package #{self.name}`

Reply:
Then I get this:

```
info esy 0.6.2 (using esy.json)
File "src/bucklescript_bin/Bin.re", line 1, characters 0-22:
Error (warning 33): unused open Migrate_parsetree.
error: command failed: 'refmterr' 'dune' 'build' '--promote-install-files' '--root' '.' '--only-package' 'graphql_ppx' (exited with 1)
esy-build-package: exiting with errors above...
error: build failed with exit code: 1
  
esy: exiting due to errors above
```

:confused: Should I remove the unused open?

Reply:
Sounds like it!